test/Base.t.sol

```diff
@@ -19,14 +20,25 @@ abstract contract Base_Test is PRBTest, StdCheats, PRBMathAssertions, PRBMathUti
         address eve;
     }
 
+    /*//////////////////////////////////////////////////////////////////////////
+                                    CHEATCODES
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @dev An instance of the Foundry VM, which contains cheatcodes for testing.
+    Vm internal constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
     /*//////////////////////////////////////////////////////////////////////////
                                      CONSTANTS
     //////////////////////////////////////////////////////////////////////////*/
 
+    int256 internal constant MAX_INT256 = type(int256).max;
+
     uint128 internal constant MAX_UINT128 = type(uint128).max;
 
     uint128 internal constant MAX_UINT40 = type(uint40).max;
 
+    int256 internal constant MIN_INT256 = type(int256).min;
```

Comment:
Let's order this alphabetically.

Reply:
> Why? Does Forge Std no longer provide them?

Forge std never provided them. They were [defined in PRBTest](https://github.com/PaulRBerg/prb-test/blob/bdc94959599f18ff135e54615e2eb02b93abba63/src/PRBTest.sol#L60C1-L67C60).

> Let's order this alphabetically.

It's already ordered alphabetically.
- MAX_INT256
- MAX_UINT128
- MAX_UINT40
- MIN_INT256

Reply:
> Forge std never provided them. They were defined in PRBTest

Oooh

> It's already ordered alphabetically.

Missed the `MIN` prefix!